specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/examples/ApplicationGatewayPrivateEndpointConnectionDelete.json

```diff
@@ -0,0 +1,14 @@
+{
+  "parameters": {
+    "api-version": "2020-04-01",
```

Comment:
Wrong API version in this and other examples

Reply:
Sure thanks for catching. I missed these mainly as I was not able to commit changes in April release. 
I have addressed the comment